ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/system-control-plane.j2

```diff
@@ -0,0 +1,38 @@
+{# eos - system #}
+{% if system.control_plane is arista.avd.defined %}
+!
+system control-plane
+{# control_plane tcp mss ceiling #}
```

Comment:
This could be simplified with:
```jinja2
{# control_plane tcp mss ceiling #}
{%    if system.control_plane.tcp_mss.ipv4 is arista.avd.defined or system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
{%        set cp_mss_cli = "tcp mss ceiling" %}
{%        if system.control_plane.tcp_mss.ipv4 is arista.avd.defined %}
{%           set cp_mss_cli = cp_mss_cli ~ " ipv4 " ~ system.control_plane.tcp_mss.ipv4 %}
{%        endif %}
{%        if system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
{%           set cp_mss_cli = cp_mss_cli ~ " ipv6 " ~ system.control_plane.tcp_mss.ipv6 %}
{%        endif %}
  {{ cp_mss_cli }}
{%    endif %}
```

Reply:
Optimized as suggested and added " " to prevent ipv4 + ipv6  erroneously sticking together without appropriate spacing when configured together:
```jinja2
{%     if system.control_plane.tcp_mss.ipv4 is arista.avd.defined or system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
{%         set cp_mss_cli = "tcp mss ceiling " %}
{%         if system.control_plane.tcp_mss.ipv4 is arista.avd.defined %}
{%             set cp_mss_cli = cp_mss_cli ~ "ipv4 " ~ system.control_plane.tcp_mss.ipv4 ~ " " %}
{%         endif %}
{%         if system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
{%             set cp_mss_cli = cp_mss_cli ~ "ipv6 " ~ system.control_plane.tcp_mss.ipv6 %}
```